reporter/reporter.go

```diff
@@ -20,6 +21,8 @@ type TestEvent struct {
 	Output  string
 }
 
+var filenameMatcher = regexp.MustCompile(`([a-zA-Z0-9_/]+\.go):(\d+)`)
```

Comment:
I think using a regex named group capture here will make the code more readable and easier to maintain over time.